user/managing-os/templates.md

```diff
@@ -31,6 +31,7 @@ We guarantee that the binary updates are compiled from exactly the same source c
 
  * [Fedora](/doc/templates/fedora/) (default base template)
  * [Fedora - Minimal](/doc/templates/fedora-minimal)
+ * [Fedora - Xfce](/doc/templates/fedora-xfce)
```

Comment:
Is `Fedora - Xfce` really an official template?

CC: @marmarek

Reply:
I'm fine with adding it as such. It's based on official Fedora template and also is regularly tested by @fepitre